spec/controllers/api/lists_controller_spec.rb

```diff
@@ -5,10 +5,11 @@
     @user = create(:user)
```

Comment:
If you want to create multiple factories, you can use `create_list` (same idea below, when creating multiple lists)